__tests__/app.test.js

```diff
@@ -147,4 +147,58 @@ describe('GET /api/articles', () => {
         });
       })  
     })
+});
+describe('GET /api/articles/:article_id/comments', () => {
+  describe('Basic request checks', () => {
+    test('returns status 200 on successful request', () => {
+        return request(app).get('/api/articles/1/comments').expect(200);
+    });
+    test('responds with an array of comment objects for a given article id, each with expected properties', () => {
+      return request(app)
+        .get('/api/articles/1/comments')
+        .then(({ body }) => {
+          const expectedProperties = [
+            'comment_id',
+            'votes',
+            'created_at',
+            'author',
+            'body',
+            'article_id'
+            ];
+          body.comments.forEach((comment) => {
+            expectedProperties.forEach(property => {
+              expect(comment).toHaveProperty(property);
+          });
```

Comment:
Nice. We want to check the values too though. Including that the article_id is the one we asked for.